src/scenarios/personalization/ArrayAddFavorites/ArrayAddFavorites.tsx

```diff
@@ -0,0 +1,245 @@
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { useState } from 'react';
+import {
+  Container,
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import { getApolloClient } from '../../../apollo-client';
+import {
+  getMoviesQuery,
+  getTvShowsQuery,
+  setArrayItemMutation,
+} from './graphql-documents';
+
+interface entity {
+  id: string;
+  title: string;
+  type: string;
+}
+
+export const ArrayAddFavorites: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [userAccessToken, setUserAccessToken] = useState<string>();
+
+  const [entityArray, setEntityArray] = useState<entity[]>([]);
+  const [entityId, setSelectedEntity] = useState<string>('');
+
+  const fetchAllCatalogItems = async (): Promise<void> => {
+    try {
+      const apolloClient = getApolloClient(
+        new URL('graphql', activeProfile.catalogServiceBaseURL).href,
+      );
+
+      const resultMovies = await apolloClient.query({
+        query: getMoviesQuery,
+        context: {
+          headers: {
+            Authorization: `Bearer ${userAccessToken}`,
+          },
+        },
+        fetchPolicy: 'no-cache',
+      });
+
+      const resultTvShows = await apolloClient.query({
+        query: getTvShowsQuery,
+        context: {
+          headers: {
+            Authorization: `Bearer ${userAccessToken}`,
+          },
+        },
+        fetchPolicy: 'no-cache',
+      });
+
+      const moviesArray = resultMovies.data.movies.nodes;
+      const tvShowsArray = resultTvShows.data.tvshows.nodes;
+
+      // Add Entity Type
+      for (let i = 0; i < moviesArray.length; i++) {
+        moviesArray[i] = Object.assign(moviesArray[i], {
+          type: 'Movie',
+        });
+      }
+
+      for (let i = 0; i < tvShowsArray.length; i++) {
+        tvShowsArray[i] = Object.assign(tvShowsArray[i], {
+          type: 'TV Show',
+        });
+      }
+
+      // Concat Entity Arrays
+      const result = moviesArray.concat(tvShowsArray);
+      setEntityArray(result);
+
+      logger.log(`calling [${fetchAllCatalogItems.name}]`, 'output:', result);
```

Comment:
It's better to log the raw output from the graphql query rather than logging the processed output here